Sources/PackageDescription/Package.swift

```diff
@@ -147,3 +147,15 @@ private func dumpPackageAtExit(_ package: Package, fileNo: Int32) {
     dumpInfo = (package, fileNo)
     atexit(dump)
 }
+
+// MARK: URL Normalization
+
+extension String {
+    private func normalizedUrl() -> String {
```

Comment:
`Use the “ed/ing” rule to name the nonmutating counterpart of a mutating method, `

Ah never mind... I think you're fine.